src/test/orphanage_tests.cpp

```diff
@@ -0,0 +1,137 @@
+// Copyright (c) 2011-2022 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include <arith_uint256.h>
```

Comment:
iwyu (@fanquake) will fix this, but for completeness I wanted to mention that you partially reverted 173c79626867e9f89d49be7dcbb0c2042c480553